### ISSUE TITLE
Requirements for ui_lovelace_minimalist not found: ['aiofiles==0.8.0']

### ISSUE THREAD

**Issue by andrei4002** [UI-Lovelace-Minimalist/UI#1190]
**Describe the bug**

Trying to install UI Lovelace Minimalist and getting an error saying 

```
Requirements for ui_lovelace_minimalist not found: ['aiofiles==0.8.0']
```

This would be the full log:

```
Logger: aiohttp.server
Source: requirements.py:275
First occurred: 12:17:47 PM (1 occurrences)
Last logged: 12:17:47 PM

Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/web_middlewares.py", line 117, in impl
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 60, in security_filter_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 100, in forwarded_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 80, in ban_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 236, in auth_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 145, in handle
    result = await result
  File "/usr/src/homeassistant/homeassistant/components/config/config_entries.py", line 148, in post
    return await super().post(request)
  File "/usr/src/homeassistant/homeassistant/components/http/data_validator.py", line 72, in wrapper
    result = await method(view, request, data, *args, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/data_entry_flow.py", line 71, in post
    result = await self._flow_mgr.async_init(
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 798, in async_init
    flow, result = await task
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 816, in _async_init
    flow = await self.async_create_flow(handler, context=context, data=data)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 922, in async_create_flow
    await async_process_deps_reqs(self.hass, self._hass_config, integration)
  File "/usr/src/homeassistant/homeassistant/setup.py", line 384, in async_process_deps_reqs
    await requirements.async_get_integration_with_requirements(
  File "/usr/src/homeassistant/homeassistant/requirements.py", line 52, in async_get_integration_with_requirements
    return await manager.async_get_integration_with_requirements(domain)
  File "/usr/src/homeassistant/homeassistant/requirements.py", line 171, in async_get_integration_with_requirements
    await self._async_process_integration(integration, done)
  File "/usr/src/homeassistant/homeassistant/requirements.py", line 186, in _async_process_integration
    await self.async_process_requirements(
  File "/usr/src/homeassistant/homeassistant/requirements.py", line 246, in async_process_requirements
    self._raise_for_failed_requirements(name, missing)
  File "/usr/src/homeassistant/homeassistant/requirements.py", line 275, in _raise_for_failed_requirements
    raise RequirementsNotFound(integration, [req])
homeassistant.requirements.RequirementsNotFound: Requirements for ui_lovelace_minimalist not found: ['aiofiles==0.8.0'].
```

**Additional context**
Running Home assistant 2023.2.5 in a docker container
From what I see it already has aiofiles version 23.1.0 installed

**Comment by contagon** [comment]
I'm also hitting this error now, is there anyway we can bump aiofiles to match the version installed by default in HA? Is there any breaking change in this at all?

See [this comment in the LSIO HA docker repo](https://github.com/linuxserver/docker-homeassistant/issues/70#issuecomment-1583066989).

**Comment by andruby** [comment]
Some problem here. I can't use ui_lovelace_minimalist because it wants to install aiofiles 0.8.0, when 23.2.1 is already installed from HA.

This is really brittle. Did anyone figure out how to fix it?

In the actual error, it looks like the installation of 0.8.0 tries to uninstall 23.2.1 (which could probably break HA if it succeeded)
```
ERROR: Cannot uninstall aiofiles 23.2.1, RECORD file not found.
```

**Comment by MastrUsr** [comment]
My issue came when installing some other integration that needed a newer aiofiles..

For me the solution was to change the == to >= in this file.
.../homeassistant/custom_components/ui_lovelace_minimalist/manifest.json

From:
    "requirements": [
        "aiofiles>=0.8.0",

To:
    "requirements": [
        "aiofiles==0.8.0",


Full file:

    "codeowners": [
        "@stokkie90"
    ],
    "config_flow": true,
    "dependencies": [
        "lovelace",
        "http",
        "frontend"
    ],
    "documentation": "https://ui-lovelace-minimalist.github.io/UI/",
    "domain": "ui_lovelace_minimalist",
    "iot_class": "calculated",
    "issue_tracker": "https://github.com/UI-Lovelace-Minimalist/UI/issues",
    "name": "UI Lovelace Minimalist",
    "requirements": [
        "aiofiles>=0.8.0",
        "aiogithubapi>=22.2.4"
    ],
    "version": "v1.3.10"

**Comment by wilbiev** [comment]
Fixed in new release v1.3.11
